editor/components/rich-text/format-toolbar/index.js

```diff
@@ -221,7 +221,7 @@ class FormatToolbar extends Component {
 						<div className="editor-format-toolbar__link-container" style={ { ...focusPosition } }>
 							<Popover
 								position="bottom center"
-								focusOnMount={ !! isAddingLink }
+								focusOnMount={ !! isAddingLink ? 'firstElement' : false }
```

Comment:
The `!!` isn't necessary here since we're just using the result in a conditional.

Reply:
![iu](https://user-images.githubusercontent.com/90871/41964527-1e168aae-79f2-11e8-8825-2c0d3ad870f0.jpeg)

But yeah fair enough.